sous_chef/sous_chef/menu/create_menu.py

```diff
@@ -39,6 +40,18 @@ class MenuRecipe:
     from_recipe: str
 
 
+class MenuSchema(pa.SchemaModel):
```

Comment:
Should we create a LoadedMenuSchema that is this plus rating, total time cooked?